model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -285,6 +285,14 @@ def _get_single_model_cmd(self, index):
         if self._is_multi_model():
             cmd += ["--enable-mpi"]
         cmd += self._get_pa_cli_command(index).replace("=", " ").split()
+
+        # OPTME: There should be a more elegant way of determining how to add EOS
+        #        We have to do it here because we use a dictionary to create the PA command
+        #        and it already contains `--request-parameter`
+        if "--periodic-concurrency-range" in cmd:
+            cmd.append("--request-parameter")
+            cmd.append("ignore_eos:true:bool")
```

Comment:
Were you able to make any progress with the link that @matthewkotila gave you?

Reply:
This is a larger change than I want to do right now. It'd be nice to clean this up, but not necessary at this point.